python/distrdf/common/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ if (ROOT_test_distrdf_pyspark_FOUND AND ROOT_test_distrdf_dask_FOUND)
 
     ROOTTEST_ADD_TEST(common_test_rungraphs
                     MACRO test_rungraphs.py
-                    ENVIRONMENT ${PYSPARK_ENV_VARS})
+                    ENVIRONMENT ${PYSPARK_ENV_VARS}
```

Comment:
This is quite a few tests to run in serial mode. The problem is that they would run alone with respect to all other tests, not only the distributed RDF tests.

Could we unite all the Dask ones and all the Spark ones under one common test module each, which create one Dask client and one Spark scheduler, respectively, and run all their tests with that? Ideally we would still keep the separate modules for different tests, but have one common supermodule that imports them?

Reply:
Thanks for the hint, I will try that